doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -467,6 +467,34 @@ destroy all backtrack and return values.
 Backtracking
 ++++++++++++
 
+Earlier, we said that "the `Proofview.tactic` monad is essentially an
+IO monad together with backtracking state representing the proof
+state." What this means is that all Ltac2 expressions of type `'a`
+either fail to evaluate to a value (and raise an exception) or
+implicitly return, not just a value of type `'a`, but also an
+exception handler which will be used if a computation "downstream"
+raises an exception. This additional information is suppressed from
+the type system for readability.
+
+For example, if `t_1` is an Ltac2 expression of type `unit`, and `t_2`
+is an expression of type `'a`, then the expression `t_1; t_2` first
+evaluates `t_1` to return a value of unit type, a change in the proof
+state, and a continuation expressing how `t_2` should proceed in the
```

Comment:
@ppedrot Here and in other places I use the word "continuation" interchangeably with "exception handler", hopefully indicating to the reader that they are the same thing.